internal/servers/controller/handlers/authtokens/authtoken_service.go

```diff
@@ -143,34 +144,36 @@ func (s Service) DeleteAuthToken(ctx context.Context, req *pbs.DeleteAuthTokenRe
 }
 
 func (s Service) getFromRepo(ctx context.Context, id string) (*pb.AuthToken, error) {
+	const op = "authtokens.(Service).getFromRepo"
 	repo, err := s.repoFn()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, op)
 	}
 	u, err := repo.LookupAuthToken(ctx, id)
 	if err != nil {
 		if errors.IsNotFoundError(err) {
-			return nil, handlers.NotFoundErrorf("AuthToken %q doesn't exist.", id)
+			return nil, nil
```

Comment:
Hmm i see we are checking the `u == nil` case below as well.  We could refactor this into something like:
```
u, err := repo.LookupAuthToken(ctx, id)
if err != nil && !errors.IsNotFoundError(err) {
     return nil, errors.Wrap(err, op)
}
if u == nil {
...
}
```

Reply:
tbh, some of these I saw and **do not** remember changing but I left them in because 🤷🏻‍♀️ 
This is one of those ones (also the one above this comment)